test/integration/features/coolstore.feature

```diff
@@ -0,0 +1,137 @@
+@coolstore
+Feature: Cool Store
+  In order to test Minishift under load
+  I need to setup a test environment
+  Using CentOS and 4G of memory
+  After which I will import image streams
+  Finally I will deploy the Cool Store
+  During this deployment I intend to measure memory and CPU usage
+
+  Scenario: Enabling xpaas addon
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift addons enable xpaas"
+     Then stdout should contain
+      """
+      Addon 'xpaas' enabled
+      """
+
+  Scenario: Starting Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --memory 4096 --iso-url centos" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: User can create new namespace coolstore
+   Given Minishift has state "Running"
+    When executing "oc new-project coolstore" succeeds
+    Then stdout should contain
+     """
+     Now using project "coolstore"
+     """
+  
+  Scenario: User can create a template for coolstore 
+   Given Minishift has state "Running"
+    When executing "oc create -f $(template_dir)/coolstore.yaml" succeeds
```

Comment:
Sure, I kind of expected this to be defined somewhere, but it comes a bit out of the blue. 

@agajdosi and I had a discussions around using variables with the go-dog maintainer. Using variables the way we do now in this pull request was already once on the table when @agajdosi was making some changes -https://github.com/DATA-DOG/godog/issues/81

From a Cucumber perspective it is not recommended to do it. So far we did not introduce it and @agajdosi solved his problem in another way. Personally I also feel that variable interpolation (or what you want to call it now) is useful and eventually needed. There is a difference so between the explicit 'set variable' step and this "magic" $(template_dir). Why not specify a relative path?